### PR TITLE
Add pm.mirko.Amusiz

### PR DESCRIPTION
**Pull request by mirkobrombin**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build  pm.mirko.Amusiz

**Comment by mirkobrombin** [comment]
> Ignoring bot build request due to repo lookup error: Invalid id flathub.

I forgot something?

**Comment by hfiguiere** [comment]
bot, build pm.mirko.Amusiz

**Comment by hfiguiere** [comment]
> > Ignoring bot build request due to repo lookup error: Invalid id flathub.
> 
> I forgot something?

Just the bot that has hiccups

**Comment by mirkobrombin** [comment]
> > > Ignoring bot build request due to repo lookup error: Invalid id flathub.
> > 
> > 
> > I forgot something?
> 
> Just the bot that has hiccups

Legit

**Comment by mirkobrombin** [comment]
```
AppData problem: tag-missing : <description> required
Error loading AppData file: AppData file /app/share/appdata/pm.mirko.Amusiz.appdata.xml was not valid
```

Maybe I didn't understand the issue but the description is there https://github.com/mirkobrombin/Amusiz/blob/92b5e4d6805b2a0b3bd83a4f12e55d77a120cb28/data/pm.mirko.Amusiz.appdata.xml.in#L9

**Comment by mirkobrombin** [comment]
bot, build pm.mirko.Amusiz

File: pm.mirko.Amusiz.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- "--socket=wayland"
+- "--env=IS_FLATPAK=1"
+modules:
+- name: amusiz
+  builddir: true
+  buildsystem: meson
+  sources:
+  - type: git
+    url: https://github.com/mirkobrombin/Amusiz
+    tag: 1.3.2
```
Comment (hfiguiere) [inline pm.mirko.Amusiz.yml:20]: Plase add a commit aswell. To protect against retagging.

Comment (mirkobrombin) [reply pm.mirko.Amusiz.yml:20]: can I put both tags and commits in the same module?

**Comment by mirkobrombin** [comment]
Now everything should be fine. I have tested all Amusiz features with the latest build (52996) and everything works fine.

**Comment by mirkobrombin** [comment]
bot, build pm.mirko.Amusiz

File: pm.mirko.Amusiz.yml (hunk 1 above)
Comment (hfiguiere) [reply pm.mirko.Amusiz.yml]: yes.

**Comment by mirkobrombin** [comment]
I am forced to freeze this PR, we are facing problems with the latest Amazon Music updates and I may need some time to fix them

**Pull request closed without merging**